magento/magento-functional-tests-migration
# Convert CreateCatalogPriceRuleEntityTest to MFTF #232

### Description (*)
Convert CreateCatalogPriceRuleEntityTest to MFTF

### Fixed Issues (if relevant)

1. magento/magento-functional-tests-migration#232: Convert CreateCatalogPriceRuleEntityTest to MFTF

### Manual testing scenarios (*)
1. Log to Admin
2. Go to Catalog price rule page
3. Create new Inactive rule
4. Check in grid
5. Delete the rule

### Contribution checklist (*)
 - [*] Pull request has a meaningful description of its purpose
 - [*] All commits are accompanied by meaningful commit messages
